test/functional/app/api/test_ussd_api.py

```diff
@@ -127,11 +141,66 @@ def test_golden_path_send_token(mocker, test_client,
 
 
 def test_invalid_service_code(mocker, test_client,
-                                init_database, initialised_blockchain_network, init_seed):
-
+                              init_database, initialised_blockchain_network, init_seed):
     org = OrganisationFactory()
     sender = UserFactory(preferred_language="en", phone=make_kenyan_phone(phone()), first_name="Bob", last_name="Foo",
-        pin_hash=User.salt_hash_secret('0000'), default_organisation=org)
+                         pin_hash=User.salt_hash_secret('0000'), default_organisation=org)
 
-    resp = req("", test_client, sender, '*42*666#')
+    resp = req("", test_client, sender.phone, '*42*666#')
     assert 'END Bonyeza {} kutumia mtandao'.format(valid_service_code) in resp
+
+
+@pytest.fixture(scope='module')
+def create_temporary_user(test_client, init_database, create_organisation):
+    # create organisation
+    organisation = create_organisation
+    organisation.external_auth_password = config.EXTERNAL_AUTH_PASSWORD
+
+    # set active organisation
+    g.active_organisation = organisation
+
+    # create user without a transfer account
+    user = create_user_without_transfer_account(unregistered_user_phone)
```

Comment:
I see we have no test for the "attach transfer account" hack from last time. Could we have one please?